spring-integration-debezium/src/main/java/org/springframework/integration/debezium/inbound/DefaultDebeziumHeaderMapper.java

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright 2023-2023 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.integration.debezium.inbound;
+
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import io.debezium.engine.Header;
+
+import org.springframework.messaging.MessageHeaders;
+import org.springframework.messaging.support.HeaderMapper;
+import org.springframework.util.Assert;
+import org.springframework.util.CollectionUtils;
+import org.springframework.util.PatternMatchUtils;
+
+/**
+ * Specifies how to convert Debezium {@link ChangeEvent} {@link Header}s into {@link Message} headers.
+ *
+ * @author Christian Tzolov
+ * @since 6.2
+ */
+public class DefaultDebeziumHeaderMapper implements HeaderMapper<List<Header<Object>>> {
+
+	public static final String DEBEZIUM_INBOUND_HEADER_NAME_PATTERN = "DEBEZIUM_INBOUND_HEADERS";
+
+	private static final String[] DEBEZIUM_HEADER_NAMES = {
+			DebeziumHeaders.DESTINATION,
+			DebeziumHeaders.KEY
```

Comment:
But as we know from channel adapter impl that these props are not header in a `ChangeEvent`.
So, it useless to look for them here the logic of this class.